README.md

```diff
@@ -38,3 +38,29 @@ yarn bundle
 ```
 
 The resulting bundles can be found within the `dist` directory at the root of the repo.
+
+## Storybook
+
+The command to run the storybook application is:
+```
+yarn storybook
+```
+
+To create a new story, in the component directory create a new directory called `stories/` and in that directory write the stories in `index.ts`.
+
+```
+/packages/web-components/@sfx/
+|-- component
+|   |-- src
+|   |   |-- index.ts
+|   |-- stories
+|   |   |-- index.ts
+|   |   |-- README.md (optional) **WIP** 
+```
+`/* @todo Make README.md default to the notes tab if file is present */`
```

Comment:
Is this TODO complete? I do see the README in the notes tab.

Reply:
No not yet, the README you see in the notes tab is what’s hard coded into the story file. I can spend an hr doing this todo but I wanted to get the PR in.